mozilla-services/tecken
# bug 1672712: update react-scripts

This updates node to 15.7.0-slim (for both Docker images) and updates react-scripts to 4.0.1. It also updates prettier, mobx, and mobx-react to current versions. Plus it rebuilds `yarn.lock`.

To test:

1. run `make run` which launches the dev frontend
2. click around and make sure the dev frontend is working
3. run `make lint` and `make lintfix` to make sure the new version of prettier doesn't have changes it wants to make
